kyverno-policy-deployment/policy-deployment.md

```diff
@@ -0,0 +1,79 @@
+# Handling Kyverno Policy Deployment
```

Comment:
This page is moved without modification. It was previously the README. See sharing-policies for substantive change